### PR TITLE
Add send "poll response" and send "poll end" to UniFFI

### PR DESCRIPTION
**Pull request by alfogrillo**
Fix https://github.com/matrix-org/matrix-rust-sdk/issues/2471
Fix https://github.com/matrix-org/matrix-rust-sdk/issues/2470

### CODE REVIEW

File: bindings/matrix-sdk-ffi/src/room.rs [hunk 1]
```diff
@@ -449,6 +452,59 @@ impl Room {
         Ok(())
     }
 
+    pub fn send_poll_vote(
+        &self,
+        answers: Vec<String>,
+        poll_start_id: String,
```
[the remark this answers is not in the document]
Comment (alfogrillo) [reply bindings/matrix-sdk-ffi/src/room.rs:457]: Applied -> https://github.com/matrix-org/matrix-rust-sdk/pull/2473/commits/aa5794494f4c931c97755012a150097a303314bf

[the remark this answers is not in the document]
Comment (alfogrillo) [reply bindings/matrix-sdk-ffi/src/room.rs:457]: (but keeping the same name `poll_start_id` as ruma is doing)

File: bindings/matrix-sdk-ffi/src/room.rs [hunk 2]
```diff
@@ -449,6 +452,59 @@ impl Room {
...
+            Some(t) => Arc::clone(t),
+            None => {
+                return Err(anyhow!("Timeline not set up, can't send the poll vote").into());
+            }
+        };
+
+        let poll_start_event_id: &EventId =
+            poll_start_id.as_str().try_into().context("Failed to create EventId.")?;
+        let poll_response_event_content =
+            UnstablePollResponseEventContent::new(answers, poll_start_event_id.to_owned());
```
Comment (jplatte) [inline bindings/matrix-sdk-ffi/src/room.rs:471]:
```suggestion
            UnstablePollResponseEventContent::new(answers, poll_start_event_id);
```

File: bindings/matrix-sdk-ffi/src/room.rs [hunk 3]
```diff
@@ -449,6 +452,59 @@ impl Room {
...
+            Some(t) => Arc::clone(t),
+            None => {
+                return Err(anyhow!("Timeline not set up, can't end the poll").into());
+            }
+        };
+
+        let poll_start_event_id: &EventId =
+            poll_start_id.as_str().try_into().context("Failed to create EventId.")?;
+        let poll_end_event_content =
+            UnstablePollEndEventContent::new(text, poll_start_event_id.to_owned());
```
Comment (jplatte) [inline bindings/matrix-sdk-ffi/src/room.rs:498]: Same here.

Comment (alfogrillo) [reply bindings/matrix-sdk-ffi/src/room.rs]: Applied: https://github.com/matrix-org/matrix-rust-sdk/pull/2473/commits/8d871e177c1d416e99c0ebe2449e0647f550e785

File: bindings/matrix-sdk-ffi/src/room.rs (hunk 2 above)
Comment (alfogrillo) [reply bindings/matrix-sdk-ffi/src/room.rs]: Applied: https://github.com/matrix-org/matrix-rust-sdk/pull/2473/commits/8d871e177c1d416e99c0ebe2449e0647f550e785

File: bindings/matrix-sdk-ffi/src/room.rs [hunk 4]
```diff
@@ -449,6 +452,59 @@ impl Room {
...
+    ) -> Result<(), ClientError> {
+        let timeline = match &*RUNTIME.block_on(self.timeline.read()) {
+            Some(t) => Arc::clone(t),
+            None => {
+                return Err(anyhow!("Timeline not set up, can't send the poll vote").into());
+            }
+        };
+
+        let poll_start_event_id: &EventId =
+            poll_start_id.as_str().try_into().context("Failed to create EventId.")?;
```
[the remark this answers is not in the document]
Comment (alfogrillo) [reply bindings/matrix-sdk-ffi/src/room.rs]: Applied https://github.com/matrix-org/matrix-rust-sdk/pull/2473/commits/8d871e177c1d416e99c0ebe2449e0647f550e785

**Pull request merged**